src/locales/en.json

```
@@ -448,45 +448,113 @@
+      "VoteTextZero": "You need to be {token} holder to initiate a blockchain vote for releasing the accumulated spread amount of {spread} tokens held by the smart contract and transfering it to æternity blockchain address by your choice.",
       "ConfirmMessage": "Please confirm the transaction!",
       "CreateVote": {
+        "YourMessage": "Your message to the community",
         "Placeholder": "Tell the community why you need the funds to be released",
+        "AddressPlaceholder": "Enter blockchain address or .chain name",
         "ProgressMessage": [
           "Creating voting contract... (step 1 of 2)",
           "Adding your vote to WordBazaar... (step 2 of 2)"
         ]
       },
       "Title": "{word}: Word Details",
-      "Abbreviation": "Abbreviation",
-      "CurrentPrice": "Current Price",
-      "SellPrice": "Sell Price",
-      "Supply": "Circulating Supply",
-      "Asset": "Asset",
+      "Abbreviation": "Short name",
+      "CurrentPrice": "Buy price",
+      "SellPrice": "Sell price",
+      "Supply": "Current supply",
+      "Asset": "Token",
       "Description": "Description",
       "BondingCurve": "Bonding Curve",
-      "LegendBuyPrice": "Bonding Buy Price",
-      "LegendSellPrice": "Bonding Sell Price",
+      "PriceHistory": "Price History",
+      "LegendBuyPrice": "Buy price",
+      "LegendSellPrice": "Sell price",
+      "LegendBondingBuyPrice": "Bonding Buy Price",
+      "LegendBondingSellPrice": "Bonding Sell Price",
       "LegendSupply": "Current Supply",
       "LegendInitialPrice": "Initial Buy Price",
       "LegendSpread": "Spread",
       "LegendReserve": "Reserve",
       "Release": "Release {0} spread for transfer to",
+      "BuyMobile": "Buy",
+      "Buy": "Buy tokens",
+      "BuyMore": "Buy more tokens",
       "Initiate": "Initiate Vote",
-      "Spread": "Accumulated Spread",
-      "Stake": "Majority Stake",
-      "RibbonTabs": ["Token Info", "Funds release vote"],
-      "Tabs": ["Ongoing Votes", "Past Votes", "My Votes"]
+      "InitiateMobile": "Initiate",
+      "Spread": "Accumulated spread",
+      "Stake": "Majority stake",
+      "RibbonTabs": [
+        {
+          "activity": "info",
+          "text": "Token info",
+          "textMobile": "Info"
+        },
+        {
+          "activity": "voting",
+          "text": "Funds release vote",
+          "textMobile": "Vote"
+        }
+      ],
+      "Tabs": [
+        {
+          "tab": "ongoing",
+          "text": "Ongoing Votes"
+        },
+        {
+          "tab": "past",
+          "text": "Past Votes"
+        },
+        {
+          "tab": "my",
+          "text": "My Votes"
+        }
+      ],
+      "TimeScopeTabs": [
+        { "text": "Day", "tab": "1" },
+        { "text": "Week", "tab": "7" },
+        { "text": "Month", "tab": "30" },
+        { "text": "3 Months", "tab": "90" },
+        { "text": "Year", "tab": "365" },
+        { "text": "All", "tab": "0" }
```

```suggestion
        "day": "Day",
        "week": "Week",
        "month": "Month",
        "3-month": "3 Months",
        "year": "Year",
        "all": "All",
```
please, don't extract configuration to localization files, we don't need the ability to change it for different locales